test/v2_integration.py

```diff
@@ -934,9 +934,9 @@ def test_new_order_policy_errs():
     except messages.Error as e:
         ok = True
         if e.typ != "urn:ietf:params:acme:error:rejectedIdentifier":
-            raise(Exception('Expected rejectedIdentifier type problem, got {0}'.format(e.typ)))
-        if e.detail != 'Error creating new order :: Policy forbids issuing for "out-addr.in-addr.arpa" and 1 more identifiers. Refer to sub-problems for more information':
-            raise(Exception('Order problem detail did not match expected'))
+            raise Exception('Expected rejectedIdentifier type problem, got {0}'.format(e.typ))
```

Comment:
Looks like a bad merge here - you need a parenthesis after raise now that we're on py3.